cmd/tailscale/cli/cli_test.go

```diff
@@ -810,6 +810,9 @@ func TestPrefFlagMapping(t *testing.T) {
 		case "Egg":
 			// Not applicable.
 			continue
+		case "RunWebClient":
+			// Currently behind a feature flag.
```

Comment:
Maybe link our gh issue? So we remember to eventually remove this comment

Reply:
done